oktapam/resource_server_checkout_settings.go

```diff
@@ -221,16 +218,14 @@ func (r *serverCheckoutSettingsResource) Delete(ctx context.Context, req resourc
 	}
 	defaultCheckoutDurationInSeconds := int32(900)
```

Comment:
I'm almost certain this is the wrong way to do delete, but I'm not touching it.

Reply:
There is no delete endpoint for settings resource. So, it just update the setting to default values and disable it. It's kind of same as what's done for password_settings resource - https://github.com/okta/terraform-provider-oktapam/blob/24d3620f9cca314e1b18ca83ed6bdf92b03c2648/oktapam/resource_password_settings.go#L186-L185. Having some comment here will be better for clarity.

Reply:
If we ever redo this, it'd be nice of "checkout settings" were an entity that just got attached to resources or projects or whatnot so it can be manipulated individually.

Reply:
You mean manage "checkout settings" as separate entity and then refer in project. It has include_list and exclude_list too which will be reference to resources applicable to the project. I feel in that case too will end up having separate settings for each resource.